forge-gui/res/cardsfolder/upcoming/vanish_into_eternity.txt

```diff
@@ -0,0 +1,8 @@
+Name:Vanish into Eternity
+ManaCost:2 W
+Types:Instant
+S:Mode$ RaiseCost | ValidCard$ Card.Self | Type$ Spell | Amount$ CostRaise | Relative$ True | EffectZone$ All | Description$ This spell costs {3} more to cast if it targets a creature.
```

Comment:
shouldn't it be easier way to code this? I know that the other cards are like this too

but shouldn't be a Condition Present work too?

Reply:
Right now the functionality for static ability condition checks is not as robust as spellability condition checks... static stuff is just `IsPresent$` which doesn't yet look at specific defined, etc.

I agree that this could be improved

Reply:
imo ValidTarget$ should work already

Reply:
tried it like this, but it would raise the cost regardless of target::A:SP$ ChangeZone | Origin$ Battlefield | Destination$ Exile | ValidTgts$ Permanent.nonLand | RememberTargets$ True | TgtPromt$ Select target nonland permanant | StackDescription$ SpellDescription | SpellDescription$ Exile target nonland permanent.
S:Mode$ RaiseCost | ValidCard$ Card.Self | Type$ Spell | Amount$ 3 | ConditionDefined$ Remembered | ConditionPresent$ Creature | ConditionCompare$ GE1 | Relative$ True | EffectZone$ All | Description$ This spell costs {3} more to cast if it targets a creature.

Reply:
in the meantime, the card works as scripted, proposed changes to engine are far beyond my abilities, and the PR should be allowed to be merged.